### ISSUE TITLE
Using INPUT# on a RANDOM file causes a crash

### ISSUE THREAD

**Issue by robhagemans** [robhagemans/pcbasic#23]
Reported by Steve https://sourceforge.net/p/pcbasic/discussion/bugs/thread/98880915/

**Comment by robhagemans** [comment]
```
ERROR: Traceback (most recent call last):
File "E:\branch1508\packaging\windows\build\installer\out00-PYZ.pyz\pcbasic.run", line 66, in run_once
File "E:\branch1508\packaging\windows\build\installer\out00-PYZ.pyz\pcbasic.statements", line 108, in parse_statement
File "E:\branch1508\packaging\windows\build\installer\out00-PYZ.pyz\pcbasic.statements", line 1979, in exec_input
File "E:\branch1508\packaging\windows\build\installer\out00-PYZ.pyz\pcbasic.devices", line 403, in read_var
File "E:\branch1508\packaging\windows\build\installer\out00-PYZ.pyz\pcbasic.devices", line 423, in _input_entry
File "E:\branch1508\packaging\windows\build\installer\out00-PYZ.pyz\pcbasic.devices", line 411, in _skip_whitespace
TypeError: 'in <string>' requires string as left operand, not bytearray
```


```
10 OPEN "r",1,"test"
20 PRINT #1,"Hello"
30 PUT 1,1
40 GET 1,1
50 INPUT #1,a$
60 PRINT a$
```

**Comment by robhagemans** [comment]
Crash fixed by 49058c3
